.profile.khan

```diff
@@ -99,6 +99,7 @@ if [ `uname -s` = Darwin ]; then
     export LDFLAGS="-L/usr/local/opt/openssl/lib $LDFLAGS"
     export CPPFLAGS="-I/usr/local/opt/openssl/include $CPPFLAGS"
     export CFLAGS="-I/usr/local/include -L/usr/local/lib $CFLAGS"
+    export CGO_CPPFLAGS="-Wno-error -Wno-nullability-completeness -Wno-expansion-to-defined -Wno-builtin-requires-header" $CGO_CPPFLAGS
```

Comment:
(btw, I believe it's only `-Wno-error` that is needed, since that's what promotes warnings to errors.  The rest just affect how noisy the compiler is.)

Reply:
Will add a comment. And yeah, I kind of suspect that it's the `no-error` that's significant, but I can't actually replicate the problem locally, and this was the solution provided that actually worked. Rather than spending a decent chunk of time for both myself and the handful of engineers who are able to reproduce this by running and rerunning the makefile options with different options, this seems pretty safe to just blanket reduce the warnings.